Client/src/components/common/Sidebar/index.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import './index.css'
+import Linklst from '../LinkList';
+const links = [[['Home'],['Course'], ['Student']]];
+
+export default function Sidebar() {
+  return (
+    <div className='sidebar'>
+      <img src='https://i.imgur.com/m6QdXqO.png' alt='AMDIEAST logo' className='img' width='200px' />
```

Comment:
remove generic class name AGAIN
```suggestion
      <img src='https://i.imgur.com/m6QdXqO.png' alt='AMDIEAST logo' className='sidebar-desktop-logo' width='200px' />
```